ynr/apps/elections/uk/management/commands/uk_create_joint_pseudo_parties.py

```diff
@@ -31,14 +29,11 @@ def create_or_update_party(joint_party_name, sub_parties):
     if len(registers) > 1:
```

Comment:
Just above here there is a block

```py
party_ids = sorted(
    extract_number_from_id(p.extra.slug) for p in sub_parties
)
registers = {p.extra.register for p in sub_parties}
```

should we also remove the `.extra` call from there?